spec/models/tag_spec.rb

```diff
@@ -229,6 +229,7 @@
 
       antifollowed = user.follow(antifollowed_tag)
       antifollowed.update explicit_points: -5
+      antifollowed.update implicit_points: 6
```

Comment:
Could `update(explicit_points: -5, implicit_points: 6)` at once, so optimize a bit?

Reply:
Great suggestion, my tunnel vision stopped me from realizing this, thank you will update accordingly!